<?php

namespace ContainerIwAkOJC;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/*
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class getSuluContact_Command_RecoverService extends App_KernelProdContainer
{
    /*
     * Gets the private 'sulu_contact.command.recover' shared service.
     *
     * @return \Sulu\Bundle\ContactBundle\Command\AccountRecoverCommand
     */
    public static function do($container, $lazyLoad = true)
    {
        include_once \dirname(__DIR__, 5).'/vendor/symfony/console/Command/Command.php';
        include_once \dirname(__DIR__, 5).'/vendor/sulu/sulu/src/Sulu/Bundle/ContactBundle/Command/AccountRecoverCommand.php';

        $container->privates['sulu_contact.command.recover'] = $instance = new \Sulu\Bundle\ContactBundle\Command\AccountRecoverCommand(($container->services['doctrine.orm.default_entity_manager'] ?? $container->getDoctrine_Orm_DefaultEntityManagerService()), ($container->services['sulu.repository.account'] ?? $container->getSulu_Repository_AccountService()));

        $instance->setName('sulu:contacts:accounts:recover');

        return $instance;
    }
}
